app/src/pages/inside/stepPage/modals/makeDecisionModal/copyFromHistoryLine/copyFromHistoryLine.jsx

```diff
@@ -33,25 +34,48 @@ export const CopyFromHistoryLine = ({
   modalState,
   itemData,
   setModalState,
-  collapseTabsExceptCurr,
   windowSize,
   eventsInfo,
+  activeProject,
 }) => {
-  const { formatMessage } = useIntl();
+  const [composedItems, setComposedItems] = useState(items);
   const { trackEvent } = useTracking();
 
+  useEffect(() => {
+    const itemIds = items.map((item) => item.id);
+    fetch(URLS.bulkLastLogs(activeProject), {
+      method: 'post',
+      data: { itemIds, logLevel: 'ERROR' },
+    })
+      .then((resp) => {
+        const itemsWithLogs = items.map((item) => {
+          return {
+            ...item,
+            logs: resp[item.id],
+          };
+        });
+        setComposedItems(itemsWithLogs);
+      })
+      .catch((e) => console.error(e));
+  }, []);
+
+  const source = modalState.historyChoice;
+
   const selectHistoryLineItem = (itemId) => {
-    if (itemId && itemId !== modalState.source.id) {
-      const historyItem = items.find((item) => item.id === itemId);
+    if (itemId && itemId !== source.id) {
+      const historyItem = composedItems.find((item) => item.id === itemId);
```

Comment:
I think you can find `historyItem` through the `items` directly, there is no need to select item together with logs.